app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -198,6 +201,7 @@ public Message(ZulipApp app, JSONObject message,
 
         this.setTimestamp(new Date(message.getLong("timestamp") * 1000));
         this.setMessageRead(false);
+        this.setMessageStar(false);
```

Comment:
I would suggest do something like https://github.com/zulip/zulip-android/pull/283/files#diff-75bd922e0a2c7b99434c4aa001aafc42R117 
to avoid a null case for the value! 
`@DatabaseField(columnName = MESSAGE_READ_FIELD, defaultValue = "0")`